### PR TITLE
feat(backend): Add issuer to UserCredentials

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

It's useful to remember who have the credentials in case the same credential type has multiple issuers.

# Changes

* Add field `issuer` to `UserCredentials`.

# Tests

* Tests cover new field because they check all fields.

### CODE REVIEW

**Comment by anedos-dfinity** [comment]
Good idea!

File: src/shared/src/types.rs [hunk 1]
```diff
@@ -132,6 +132,7 @@ pub mod user_profile {
     pub struct UserCredential {
         pub credential_type: CredentialType,
         pub verified_date_timestamp: Option<Timestamp>,
+        pub issuer: String,
```
Comment (peterpeterparker) [inline src/shared/src/types.rs:135]: Shouldn't this information be a variant?

Comment (lmuntaner) [reply src/shared/src/types.rs:135]: This comes from the config, the `issuer_origin` of `SupportedCredential`. There it's a String, and I believe that's ok.

I don't think we need to create each issuer into an entry in an enum.

Comment (peterpeterparker) [reply src/shared/src/types.rs:135]: Analytics might need manual work, and we won't be able to accurately use the information in the frontend for a feature. Those were my spontaneous thoughts for my question, but okay.

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

**Pull request merged**